### PR TITLE
FEAT: added hyperparam tuning of pretraining task using optuna; updat…

### PR DESCRIPTION
**Pull request by b-elfo**
…ed parameters in main.py and retrained downstream task to gain 15% improvement (now at 50% accuracy of CIFAR10);

### CODE REVIEW

File: downstream.py [hunk 1]
```diff
@@ -276,11 +276,11 @@ def validate_task(dataset_path: str,
 
     train_task(dataset_path           = 'CIFAR10',
                pretrained_weight_path = './models/CIFAR10/pretrain_model_30.pth',
-               init_lr                = 5e-3,
-               batch_size             = 16,
+               init_lr                = 5e-4,
+               batch_size             = 32,
```
Comment (b-elfo) [inline downstream.py:280]: 'Optimal' params, as returned by Optuna;
